scripts/openstack_compute_install.sh

```diff
@@ -8,6 +8,9 @@ salt "cmp*" state.apply
 # Provision opencontrail virtual routers
 salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provision_vrouter.py --host_name cmp01 --host_ip 172.16.10.105 --api_server_ip 172.16.10.254 --oper add --admin_user admin --admin_password workshop --admin_tenant_name admin"
 
+# Enables eth1 interface on compute nodes
+salt 'cmp*' cmd.run "ip link set up eth1"﻿⁠⁠⁠⁠
```

Comment:
My only wild guess so far is that once you issued this command, contrail is doing something behind the scenes whcih makes some persistent changes in the system configuration